### PR TITLE
Fix missing copy

### PR DESCRIPTION
**Pull request by HarelM**
It seems that when the folders `plugins` and `platforms` do not exists (as is usually in a CI scenario) the copy script does not run. This seems like a bug in cordova, but I believe this simple workaround is currently enough.
This should address #82.

### CODE REVIEW

**Comment by HarelM** [comment]
@maintainers Anyone?

File: hooks/iosAddTarget.js [hunk 1]
```diff
@@ -316,6 +320,8 @@ module.exports = function (context) {
             if (productName.indexOf('ShareExt') >= 0) {
               buildSettingsObj['PROVISIONING_PROFILE'] = PROVISIONING_PROFILE;
               buildSettingsObj['DEVELOPMENT_TEAM'] = DEVELOPMENT_TEAM;
+              buildSettingsObj['CODE_SIGN_STYLE'] = 'Manual';
+              buildSettingsObj['CODE_SIGN_IDENTITY'] = '"iPhone Distribution"';
```
Comment (j3k0) [inline hooks/iosAddTarget.js:324]: The rest of the PR looks good to me, but this seems like the above either solves another issue or commit 7d6fa27554bb1980d3985f097462963071962701 shouldn't be part of the PR? Should it?

Comment (HarelM) [reply hooks/iosAddTarget.js:324]: Yes and no. The issue that I was trying to resolve is #82. The issue is mainly building a cordova app in a CI environment so this is needed too, I believe.
I assume that if you are working with XCode you can check the "automatically sign..." but you can't do it from a CI so I think the default should be manual.
I can change the title of the pull request if it makes any difference.
When running Cordova from command line you can specify this in the build arguments but it won't affect this extension.
Might be it should be a feature request to Cordova-ios, not sure, but I've sent them a PR for fixing the problem with provisioning profiles and haven't heard back from them so I don't know...

Comment (j3k0) [reply hooks/iosAddTarget.js:324]: OK, I wonder if `Manual` code sign style will break the build in other cases. Let's merge and see.

**Comment by HarelM** [comment]
Thanks!!
Let me know when a new version will be available in npm so I'll be able to remove the reference to my fork.
